packages/plugins/model-train/image-classification-tensorflow-model-train/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import { ImageDataset, ModelTrainType, TfJsLayersModel, ModelTrainArgsType, ImageDataLoader } from '@pipcook/pipcook-core';
+
+const boa = require('@pipcook/boa');
+const { tuple } = boa.builtins();
+const tf = boa.import('tensorflow');
+const AUTOTUNE = tf.data.experimental.AUTOTUNE;
+
+function loadImage(path: string) {
+  let image = tf.io.read_file(path);
+  image = tf.image.decode_jpeg(image, boa.kwargs({
+    channels: 3
+  }));
+  return image;
+}
+
+async function createDataset(dataLoader: ImageDataLoader, labelMap: {
+  [key: string]: number;
+}) {
```

Comment:
May be make this as type 'LabelMapType: Record<...>' is better?

Reply:
In detailed, it's `Record<string, number>`.